src/util/history.js

```diff
@@ -34,6 +34,11 @@ const getHashUrlParams = (loc) => {
     return params
 }
 
+const getHashUrlParam = (key) => {
+    const params = getHashUrlParams(history.location)
+    return params[key]
+}
+
```

Comment:
I added this convenience method for retrieving the value of a single query parameter by name. Note that since the `parseBooleans` option is enabled in `getHashUrlParams` (note the plural `s`), we get actual booleans for things like `getHashUrlParam('isPushAnalytics')` when the query param looks like this `?isPushAnalytics=true`.